hypha/apply/funds/tests/test_models.py

```diff
@@ -214,6 +214,8 @@ def submit_form(self, page=None, email=None, name=None, user=AnonymousUser(), ig
                 data[field.id] = self.email if email is None else email
             if isinstance(field.block, FullNameBlock):
                 data[field.id] = self.name if name is None else name
+            if draft:
+                data['draft'] = "Save Draft"
```

Comment:
Maybe make this possible to translate. (This is another area that really needs some work overall, a lot of text is not translatable in the Hypha app.)